atomicfu-maven-plugin/build.gradle

```diff
@@ -3,75 +3,36 @@
  */
 
 apply plugin: 'kotlin'
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
 
 apply from: rootProject.file('gradle/compile-options.gradle')
 
 ext.configureKotlin()
 
 dependencies {
-    compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    compile project(":atomicfu-transformer")
-    compile "org.apache.maven:maven-core:$maven_version"
-    compile "org.apache.maven:maven-plugin-api:$maven_version"
-    compile 'org.apache.maven.plugin-tools:maven-plugin-annotations:3.5'
+    api "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    api project(":atomicfu-transformer")
+    api "org.apache.maven:maven-core:$maven_version"
+    api "org.apache.maven:maven-plugin-api:$maven_version"
+    api 'org.apache.maven.plugin-tools:maven-plugin-annotations:3.5'
 }
 
-def pomFile = file("$buildDir/pom.xml")
+def pomFile = file("$buildDir/publications/maven/pom-default.xml")
```

Comment:
Now I rely on the path of the pom file, created with `generatePomFileForMavenPublication`

Reply:
Fixed